components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,154 @@
+  };
+
+  const showControls = !hideControls && items?.length > 1;
+
+  return (
+    <Container id={id} mt={marginTop} mb={marginBottom} className={styles.root} {...props}>
+      {heading && <Heading size={3} as={headingLevel} className={styles.heading}>{heading}</Heading>}
+      {intro && <div className={styles.intro}>{intro}</div>}
+      {showControls && (
+        <FlexBox justifyContent="end" className={styles.controls}>
+          <CtaButton
+            disabled={allExpanded}
+            variant="ghost"
+            size="small"
+            color={isDarkTheme ? 'white' : 'black'}
+            icon="plus"
+            iconProps={{ className: styles.expandAllIcon}}
+            onClick={expandAll}
+            className={styles.controlButton(isDarkTheme)}
+          >
+            Expand All
+          </CtaButton>
+          <CtaButton
+            disabled={allCollapsed}
+            variant="ghost"
+            size="small"
+            color={isDarkTheme ? 'white' : 'black'}
+            icon="minus"
+            iconProps={{ className: styles.collapseAllIcon}}
+            onClick={collapseAll}
+            className={styles.controlButton(isDarkTheme)}
+          >
+            Collapse All
+          </CtaButton>
+        </FlexBox>
+      )}
+      <ul className={styles.list}>
+        {items?.map((item, index) => (
+          <li key={item._uid} className={styles.listItem}>
+            <Heading as={itemHeadingLevel} color={isDarkTheme ? 'white' : 'black'} leading="tight" className={styles.itemHeading}>
+              <CtaButton
+                id={`button-${item._uid}`}
+                ref={index === 0 ? firstItemRef : undefined}
+                onClick={() => toggleItem(index)}
+                variant="unset"
+                color={isDarkTheme ? 'white' : 'black'}
+                aria-expanded={openItems[index] || false}
+                aria-controls={`content-${item._uid}`}
+                className={styles.button}
+              >
+                <span aria-hidden="true" className={styles.bar} />
+                {item.heading}
+                <HeroIcon icon={openItems[index] ? 'minus' : 'plus'} className={styles.circleIcon} />
+              </CtaButton>
+            </Heading>
+            <m.div
+              role="region"
+              aria-labelledby={`button-${item._uid}`}
+              id={`content-${item._uid}`}
+              aria-hidden={!openItems[index]}
+              animate={{
+                height: openItems[index] ? 'auto' : 0,
+                visibility: openItems[index] ? 'visible' : 'hidden',
+              }}
+              initial={false}
+              transition={{ duration: 0.3, ease: 'easeIn' }}
+              className={styles.contentWrapper}
+            >
+              {/* @ts-expect-error Reason: inert not fully support in React 18. Using '' sets it to true */}
+              <div className={styles.richtextWrapper} {...(!openItems[index] ? { inert: '' } : {})}>
```

Comment:
@sherakama I found a better way adding the inert attribute to make the child items un-focusable. However, React 18 doesn't fully support this attribute, so if you set `inert={true}` (or just `inert`) it doesn't get rendered in the DOM. A workaround is to set `inert=''` instead of just adding `inert` as a boolean true attribute.
https://github.com/facebook/react/issues/17157

Reply:
Neat. Thanks for sharing that learning.